compressed_archive/libarchive/libarchive-wrapper.pri

```diff
@@ -0,0 +1,28 @@
+INCLUDEPATH += $$PWD
+DEPENDPATH += $$PWD
+
+HEADERS += $$PWD/extract_delegate.h \
+           $$PWD/compressed_archive.h
+
+SOURCES += $$PWD/compressed_archive.cpp
+
```

Comment:
"win32:" check is missing - we should at least add a message that says "currently unsupported"